### ISSUE TITLE
Generator 1 conv2d (64, 64, 64) -> (3, 64, 64)

### ISSUE THREAD

**Issue by sahilkhose** [sahilkhose/StackGAN-BERT#3]
https://github.com/sahilkhose/ganctober/blob/37aa750ab95d381bb44a6654eb20174d7c98dacc/src/layers.py#L68
There is no mention of conv2d in the paper
They mention using upsampling blocks and the last upsampling block does not have a batch norm and relu activation
We can just set output channels of self.up4 to 3, to solve this problem?
I think this conv2d is used because of the keras implementation
